notebooks/llm-chatbot/llm-chatbot.ipynb

```diff
@@ -32,6 +32,7 @@
     "- [Convert model using Optimum-CLI tool](#Convert-model-using-Optimum-CLI-tool)\n",
```

Comment:
<p>Please high-light note about unapplicability AWQ and could you please provide details what do you mean under skip (there will be some warning message, there is explicit configuration that skip it or something else?).</p><p> </p><p>Also please provide details about used dataset</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2043/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Highlighted the note about skipping of the algorithm and added information on which dataset is used for calibration.

When AWQ is skipped, there will be an NNCF INFO level log message: "No matching patterns were found for applying AWQ algorithm, it will be skipped."